### PR TITLE
Add: STO Release Notes for Week of 2024-07-07

### PR DESCRIPTION
**Pull request by tejakummarikuntla**
### Preview links
- [Version 1.102.2](https://6691311a40536d22eb9a03a4--harness-developer.netlify.app/release-notes/security-testing-orchestration#version-11022)
- [STO early access features](https://6691311a40536d22eb9a03a4--harness-developer.netlify.app/release-notes/early-access#sto-early-access-features)
- [STO features promoted to GA](https://6691311a40536d22eb9a03a4--harness-developer.netlify.app/release-notes/early-access#sto-features-promoted-to-ga)

PRs must meet these requirements to be merged:

- [x] Successful preview build.
- [x] Code owner review.
- [x] No merge conflicts.
- [x] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

**Review by priteshchandaliya — DISMISSED** [review]

File: docs/security-testing-orchestration/exemptions/exemption-workflows.md [hunk 1]
```diff
@@ -148,6 +148,11 @@ This workflow requires [Security Testing SecOps](/docs/security-testing-orchestr
     - **Cancel** The request is cancelled and removed from the exemption list. If a user wants an exemption for the issue, they must file a new request. 
 
      ![](../use-sto/static/exemptions-approved.png)
+
+:::note notes
```
Comment (douglas-j-bothwell) [inline docs/security-testing-orchestration/exemptions/exemption-workflows.md:152]:
```suggestion
:::note 
```
This is only one note, hence singular

**Review by aklaft — APPROVED** [review]

File: release-notes/security-testing-orchestration.md [hunk 2]
```diff
@@ -22,6 +22,17 @@ These release notes describe recent changes to Harness Security Testing Orchestr
...
+
+### Version 1.102.2
+
+<!-- 2024-07-11 -->
+
+#### New features and enhancements
+
+- Branch Scanning in [SonarQube](/docs/security-testing-orchestration/sto-techref-category/sonarqube-sonar-scanner-reference#scan-configuration) is now Generally Available(GA), previously behind the `STO_SONARQUBE_BRANCHING` feature flag. Customers can now perform branch scans using the SonarQube step.
+- Added a new setting called `Users can approve their own exemptions.` This setting helps control whether users can approve their own exemptions, and it can be easily enabled or disabled based on their preference. Find this setting under **Exemption settings** on the **Default settings** page. This is available in the project, organization and account level settings.
+
```
Comment (douglas-j-bothwell) [inline release-notes/security-testing-orchestration.md:35]: You still need to update this topic: [Early access features](https://668fbdf116785700a55cf401--harness-developer.netlify.app/release-notes/early-access)
- Move [SonarQube branch scanning](https://668fbdf116785700a55cf401--harness-developer.netlify.app/release-notes/early-access#sonarqube-branch-scanning) out of [STO early access features](https://668fbdf116785700a55cf401--harness-developer.netlify.app/release-notes/early-access#sto-early-access-features) and into [STO features promoted to GA](https://668fbdf116785700a55cf401--harness-developer.netlify.app/release-notes/early-access#sto-features-promoted-to-ga), then update the release info as needed.

**Review by douglas-j-bothwell — APPROVED** [review]
Approved with comments.

**Comment by tejakummarikuntla** [comment]
Thank you, @douglas-j-bothwell, addressed the reviews.

**Review by aklaft — DISMISSED** [review]

**Pull request merged**